src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -191,6 +191,9 @@ def _get_version_string(name, version_dict):
         _print()
     _print("Python location '{}'".format(sys.executable))
     _print("Extensions directory '{}'".format(EXTENSIONS_DIR))
+    import os
+    if os.path.isdir(EXTENSIONS_SYS_DIR) and os.listdir(EXTENSIONS_SYS_DIR):
+        _print("Extensions system directory '{}'".format(EXTENSIONS_SYS_DIR))
```

Comment:
it's little confusing, how about direct show as `Extensions directory`

Reply:
The idea is to have no impact on users who don't use --system to install extensions. This will only be shown when users have installed at least one extension with --system.